unaligned_bam_to_bqsr/bqsr.cwl

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: 'create BQSR table'
+baseCommand: ["/usr/bin/java", "-Xmx16g", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "BaseRecalibrator"]
+arguments:
+    ["-o", { valueFrom: $(runtime.outdir)/bqsr.table },
+    "--preserve_qscores_less_than", "6",
+    "--disable_auto_index_creation_and_locking_when_reading_rods",
+    "--disable_bam_indexing",
+    "-dfrac", ".1",
+    "-L", "chr1",
```

Comment:
Merely a comment and not a request for changes. However, for WGS QC purposes there are a few places where we would only want to evaluate the autosomal chromosomes. For example, the Picard CollectWgsMetrics should take an IntervalList format of the autosomal chr only. We should discuss whether this should be an input parameter to the workflow or perhaps stored somewhere else.  Once we have multiple places where this list is used, it seems like a bad idea to duplicate the information.